.travis.yml

```diff
@@ -8,12 +8,11 @@ php:
   - nightly
 
 env:
-  - SYMFONY_VERSION=4.2.* SOLR_VERSION=8.1.1 SOLR_CLOUD=true
-  - SYMFONY_VERSION=4.2.* SOLR_VERSION=8.1.1 SOLR_CLOUD=false
-  - SYMFONY_VERSION=4.2.* SOLR_VERSION=7.7.2 SOLR_CLOUD=true
-  - SYMFONY_VERSION=4.2.* SOLR_VERSION=7.7.2 SOLR_CLOUD=false
-  - SYMFONY_VERSION=4.2.* SOLR_VERSION=6.6.6 SOLR_CLOUD=false
-  - SYMFONY_VERSION=3.3.* SOLR_VERSION=6.6.6 SOLR_CLOUD=false
```

Comment:
We should keep one test run against `SYMFONY_VERSION=3.4*` and add one against `SYMFONY_VERSION=4.4*`